config/env/default.js

```diff
@@ -28,5 +28,19 @@ module.exports = {
   sessionKey: 'sessionId',
   sessionCollection: 'sessions',
   logo: 'modules/core/client/img/brand/logo.png',
-  favicon: 'modules/core/client/img/brand/favicon.ico'
+  favicon: 'modules/core/client/img/brand/favicon.ico',
+  uploads: {
+    profileUpload: {
+      dest: './modules/users/client/img/profile/uploads/', //Profile upload destination path
+      limits: {
+        fileSize: 1*1024*1024 //Max file size in bytes
+      },
+      fileFilter: function (req, file, cb) {
+        if (file.mimetype !== 'image/png' && file.mimetype !== 'image/jpg' && file.mimetype !== 'image/jpeg' && file.mimetype !== 'image/gif') {
+            return cb(new Error('Only image files are allowed!'), false);
+        }
+        cb(null, true);
+      }
+    }
```

Comment:
I misunderstood what this fileFilter is being used for. Is this a requirement of Multer? 

Since this is on the server side, can it be a reusable function?